### ISSUE TITLE
Statista: Update based on paywalled content policy 

### ISSUE THREAD

**Issue by wolfbeast** [duckduckgo/zeroclickinfo-spice#2400]
@statista 

Using the instant answer on DDG, Statista will provide only a couple statistics, and after that all statistics results are marked as "premium exclusive", even the ones looked up before and retrieved. This is bad practice and just serves to funnel DDG users into creating accounts and becoming paid members.

Please reconsider this IA

------
IA Page:  http://duck.co/ia/view/statista

**Comment by moollaza** [comment]
@wolfbeast I'm not sure I follow, can you please clarify with an example?

You should be able to view all "free" statistics (marked in the results), as many times as you'd like. However, the reports marked as "premium" do require an account to view them.

Are you saying you're unable to view all the free content? or that there is too much "premium" content?

**Comment by wolfbeast** [comment]
I'm saying that I ran across this instant answer doing a search with the keyword "statistics" in my search terms. I used it, it sent me to statista when clicking the answer for more details, and it all came up without a mention of premium. Since there were more hits to my search terms that were relevant, I tried calling up more, and after like 2 or 3, they locked their statistics down with a "this is a premium result" message over the actual statistics.
Now that would be fine if the results I called up additionally *were* actually premium, but the first result I got was now *also* locked down, which was previously fully displayed without mention of it being a premium statistic report.

So, as an instant answer, this really is not usable, and just serves to give statista free advertising with a "demo" of the first 2 or 3 results and then locking *all* of it down for payment. This kind of thing does not belong on DDG as an IA.

**Comment by altern8tif** [comment]
@wolfbeast I don't think they "lock down" your access to free stats after a number of results.  Their service is that some basic stats are provided for free and that information is reflected in the Spice results with a Premium or Free indicator.  It just so happens that web browser statistics don't fall under their free stats cateogry.

![screen shot 2016-01-15 at 12 55 55 pm](https://cloud.githubusercontent.com/assets/5793364/12345707/7819ca74-bb87-11e5-8ace-5317dd40b7f1.png)


I see your point though.  It seems like only 7% of the stats on statista is available for free.  The rest have to be accessed via a premium account.  But I still think there is some value for users to know that these stats exist and are available (even though it comes at a price).  If they really want to, they can then purchase a premium account.

![screen shot 2016-01-15 at 12 56 44 pm](https://cloud.githubusercontent.com/assets/5793364/12345725/ac8b956c-bb87-11e5-91c9-a7f30dd4befe.png)

@moollaza Thoughts?

**Comment by wolfbeast** [comment]
OK, I see the point now it's pointed out to me. This means that in my initial search I was given access to a premium result which was subsequently locked down. This is hardly better than popping up a paywall prompt immediately.

So, you're OK with an IA which takes up the result screen for "premium" results that come at a price of $49/month, when doing a simple search for e.g. "maxthon net statistics" as a simple/common search phrase, without it actually being listed as an ad or having Statista pay for it? **hell no** I say to that!

**Instant Answers should never be free promotion for paid services.**
This IA should never have been accepted in the first place because of this. There are plenty of freely available statistics sites that can be used, instead. If Statista wants to be listed on top of page 1, they should use the normal Ad slots for that.

**Comment by zekiel** [comment]
@wolfbeast  There is no right or wrong answer --- this is an open source community and so anyone can suggest (or make) changes according to the guidelines of the project. The idea is that every Instant Answer should use the best source possible and that may be different on a per-query basis. 

This IA is not a promotion -- it is providing useful information in many cases. That does not mean, however, that another source cannot do better for parts of the query_space. Additionally, in most cases where an IA shows both free & paid content, it's possible to restrict the output to ONLY free things but our community has (many times) preferred to see all of what is available (vs. just the free stuff). 

If you know of a [good source](http://docs.duckduckhack.com/submitting/checklist.html) that can be used for Instant Answers on searches like, "maxthon net statistics" or similar -- please let our community know. The objective is to have the best IA experience possible for every query. In some cases, it happens more slowly over time, as new IAs handle specific queries better than others.

**Comment by wolfbeast** [comment]
I'm sorry but this IA is most definitely a promotion as it ceases to work after a few "demo" results. There is most definitely a right or wrong answer here. Letting a commercial service provider ride common search terms to get 93% of their presented results lead to a paywall/funnel in my book is wrong.

This has nothing to do with Community code or Open Source; it has everything to do with vetting what becomes an integral part of the search engine and preventing abuse of an open system for commercial ends that leave the end-user without results.

Please note that I am currently a DDG partner and do bring quite a bit of traffic to DDG, but if this commercial skirting around paying for ads is going to be condoned then I may have to re-evaluate my partnership.

As for a good source? http://www.infoplease.com/ perhaps, or any of the general statistics bureaus around the world, who will all have statistical data in the public domain or at the very least free to access.

**Comment by zekiel** [comment]
@wolfbeast we are taking this to heart and want to understand exactly where the discrepancy is so we're answering (or investigating) the right thing.

Is your issue with the Paid/Premium results showing at all? I want to make sure that we're not talking relevancy vs. policy. (i.e. if the Statista IA were limited to only showing free content -- does that change the situation?)
*In cases like [app search](https://duckduckgo.com/?q=productivy+apps&ia=apps&iai=SMS+Text+Messaging+from+Tablet), [streaming services](https://duckduckgo.com/?q=watch+sherlock+online&ia=watch), etc. -- the content sometimes requires a purchase or subscription*

There are many queries where the IA experience can be improved or served better by a different IA but we rely on the community to help us with that. We're not making any money off this IA and we are not preventing another IA from taking over these queries. In some cases, [statistics are handled by a different source](https://duckduckgo.com/?q=population+statistics+uganda&ia=worldbank). If we had more IAs that covered stats like that, infoplease, etc. then statista would show only when its the most optimal IA to show.

We want to have the best IA experience possible and so we're really wanting to find clarity here. Thanks for sticking with us while we dig a little deeper!

**Comment by wolfbeast** [comment]
@zekiel thanks for taking this as serious as it is. I do think this is a matter of policy, not relevancy.

If e.g. Statista were to only show immediately accessible (free) relevant answers, then there would be absolutely no problem. See below.

From a user's point of view, premium instant answers are bad:
- They use DDG to search for relevant results. DDG should provide them with answers if the searches trigger "instant answers"
- If these IAs end up being "premium" and therefore barred behind a registration or paywall, then DDG is effectively *not* providing the user an answer to their queries, and at its core failing as a search engine.

Having companies be able to completely sidestep ranking (it'll be the top presented results by design of it being an IA), ad payments (you indicate it's free for the provider), etc. by playing the community into getting premium IAs accepted will see plenty of abuse once this gets known. Statista is a good example, here, of what can be expected.

In the case of app searches or streaming services, you're not exactly looking at the same thing, I think. The answer to the user's query isn't the app; it's knowing *about* the app. Same for media. If you're searching for a specific movie, then the answer is "where can I find it" and not the content of the movie itself. Even so, this would be better served with normal search results unless the IA is actually providing different providers *where the app or media can be obtained* (e.g. looking for a specific movie popping up an IA with several known streaming services that all offer the movie -- this is still giving the user the answer they are looking for)

Whether there will be more IAs available for similar content or not is really besides the point. If you can offer a broad range of relevant results that are all "pay first", then you can make the IA equally broad and covering more terms regardless of whether you're actually giving DDG users the answers they are looking for. I hope you understand my point here.

**Now, what if premium IAs are no longer allowed?**
It would be perfectly fine if the service provider behind an IA offers more content on their own site (e.g. if a user goes there by selecting a free IA result and being offered the option to get paid service if they want more (e.g. more detailed, etc.)), and from an exposure/fairness point of view, this would give a good balance for any company:
- The benefit of this is that the company gets exposure through an IA
- The cost of this exposure is providing relevant, free content to users of DDG

...but they *must* provide the (free) answer to the terms that triggered their IA to pop up without requiring anything else from the user (and hopefully without nagging the user too much). Anything else should be considered unacceptable.

Thanks for listening.

**Comment by zekiel** [comment]
Thanks for sticking with us @wolfbeast. We're evaluating necessary updates to the policy and hope to get back to this thread shortly.

**Comment by zekiel** [comment]
We have adjusted our IA policy based on discussions in this thread and we're working with Statista to update their IA accordingly. 

@wolfbeast  @altern8tif  thank you both for the help and patience. If you'd like to follow the updates or need more clarity, we'll be using this Issue to manage the statista adjustments and I'd be happy to discuss the policy changes in detail. I've added them to the submission guidelines here: docs.duckduckhack.com/submitting/checklist.html#does-the-data-source-have-paywalled-content
